Game/Config/DefaultEngine.ini

```diff
@@ -89,4 +89,12 @@ PhysXTreeRebuildRate=10
 DefaultBroadphaseSettings=(bUseMBPOnClient=False,bUseMBPOnServer=False,bUseMBPOuterBounds=False,MBPBounds=(Min=(X=0.000000,Y=0.000000,Z=0.000000),Max=(X=0.000000,Y=0.000000,Z=0.000000),IsValid=0),MBPOuterBounds=(Min=(X=0.000000,Y=0.000000,Z=0.000000),Max=(X=0.000000,Y=0.000000,Z=0.000000),IsValid=0),MBPNumSubdivs=2)
 ChaosSettings=(DefaultThreadingModel=DedicatedThread,DedicatedThreadTickMode=VariableCappedWithTarget,DedicatedThreadBufferMode=Double)
 
+; For NFR tests, we should adjust these once we have Spatial limits. This is to make a fair comparison between a playable game (without lag + banding)
+[/Script/OnlineSubsystemUtils.IpNetDriver]
+bNoTimeouts=true
```

Comment:
This is a lot of values - you sure we need them all?

Reply:
I believe these are server settings and the ones below are for clients

Reply:
To rate-limit RPCs?

Reply:
Yeah, those numbers are distributed to players when they join a server